plugins/woocommerce-admin/client/two-column-tasks/task-list-item-two-column.tsx

```diff
@@ -26,7 +26,9 @@ export const TaskListItemTwoColumn: React.FC< TaskListItemProps > = ( {
 	trackClick,
 } ) => {
 	const { createNotice } = useDispatch( 'core/notices' );
-	const { dismissTask, undoDismissTask } = useDispatch( OPTIONS_STORE_NAME );
+	const { dismissTask, undoDismissTask } = useDispatch(
+		ONBOARDING_STORE_NAME
+	);
```

Comment:
Found a bug with the TS check. `dismissTask`  and `*Task` are in the onboarding store, not options store.

https://github.com/woocommerce/woocommerce/blob/c83dedd84999ada6a774197d26feaef6c1333426/packages/js/data/src/onboarding/actions.js#L356-L375

Reply:
Great catch! I wonder how this didn't result in an error before 🤔

Reply:
Perhaps cc @joelclimbsthings @woocommerce/mothra in case if there's anything worth looking